File: ios/RNHelloWorldManager.m
//
//  RNHelloWorldManager.m
//  Hello
//
//  Created by Pavel Vashkel on 29/11/2022.
//

#import <Foundation/Foundation.h>
#import "React/RCTViewManager.h"

@interface RCT_EXTERN_MODULE(RNHelloWorldManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(addressee, NSString)
@end

File: ios/RNHelloWorldManager.swift
//
//  RNHelloWorldManager.swift
//  Hello
//
//  Created by Pavel Vashkel on 29/11/2022.
//

import Foundation
import React

@objc(RNHelloWorldManager)
class RNHelloWorldManager : RCTViewManager {
  
  override func view() -> UIView! {
    let label = UILabel()
    label.text = "Hello World!"
    return label
  }
}